tools/downloader/topologies.py

```diff
@@ -0,0 +1,99 @@
+import os
+import argparse
+
+from pathlib import Path
+
+import common
+import downloader
+import converter
+
+class Topology:     
+    '''
+    {}
+    Model task type: {}
+
+    License: {}
+    '''
+
+    def __init__(self, model_name, *, precision=None, download_dir=None):
+        '''
+        Downloads target model. If the precision doesn't set, loads all the available models.
+        If there is already downloaded model - retrieves the model from cache.
+        Creates an object which stores information about the downloaded model.
+
+        Parameters
+        ----------
+            model_name
+                Target model name.
+            precision
+                Target model precision. By default is None.
```

Comment:
It seems short-sighted to only allow either one or all precisions to be downloaded. Why not accept a list of precisions?

Reply:
But then which one precision value should be as ```self.precision```. Or for ```get_ir()``` we must always set precision value?

Reply:
@likholat let FP32 be a default precision in `get_ir()` method.

Reply:
@eizamaliev some intel models doesn't support ```FP32``` precision, for example ```bert-large-uncased-whole-word-masking-squad-int8-0001``` model.